internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -326,6 +328,7 @@ type image struct {
 type Input struct {
 	Attestations []attestationData `json:"attestations"`
 	Image        image             `json:"image"`
+	AppSnapshot  app.SnapshotSpec  `json:"appSnapshot"`
```

Comment:
Makes it more alike the other fields

```suggestion
	AppSnapshot  app.SnapshotSpec  `json:"snapshot"`
```

Reply:
Done